test_suite/util.sh

```diff
@@ -7,6 +7,10 @@ RED="\033[0;31m"
 GREEN="\033[0;32m"
 NC="\033[0m" # No color
 
+# Frequently used regular expressions
+int_regex="^[0-9]+$"
+real_regex="[0-9]+[.]?([0-9]+)?" # Allows for counting the amount of digits after the decimal point
```

Comment:
dit match't `0.`, is dat oke?

```suggestion
real_regex="[0-9]+([.][0-9]*)?" # Allows for counting the amount of digits after the decimal point
```

het bovenstaande matcht `0.0`, `0.`, en `0`

als je `0.` niet wil hebben, verander die laatste `*` in een `+`

Reply:
```suggestion
real_regex="[0-9]+([.][0-9]+)?" # Allows for counting the amount of digits after the decimal point
```

Zonder de 0. is inderdaad beter denk ik